ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -8,7 +8,7 @@
 {% for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
 {%     if ethernet_interfaces[ethernet_interface].channel_group.id is defined and ethernet_interfaces[ethernet_interface].channel_group.id is not none %}
 {%         set port_channel_interface = 'Port-Channel' + ethernet_interfaces[ethernet_interface].channel_group.id | string %}
-{%         if port_channel_interfaces[port_channel_interface].type is not defined or port_channel_interfaces[port_channel_interface].type != "routed" %}
+{%         if port_channel_interfaces[port_channel_interface].type is not defined or port_channel_interfaces[port_channel_interface].type in ["switched", "l2dot1q"]  %}
```

Comment:
Not sure why, but this change removes switchport from many of the molecule configs. I don't think that is on purpose.

Reply:
Ok I am looking at why. But as it is on documentation template, I don't think it is the root cause of the change you saw in CLI output.

probaably comes from this line instead: https://github.com/titom73/ansible-avd/blob/6c201e7df7a6e5db599fb3e60a8b9a7d354873da/ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2#L35

I considered type would be always defined but I will add a default statement to fallback to interface_defaults or `switched`

Reply:
Fixed with commit 988df1e and artifacts updated. Let me know your thoughts about update